CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx

```diff
@@ -179,6 +194,19 @@ export default function CreateReport({ onDone }) {
               ))}
             </select>
 
+            <label htmlFor='attributes' className="font-bold ml-4">Select Attributes to Compare:</label>
+            <select className="border border-black rounded-sm bg-gray-100 text-left  ml-4 w-[150px] h-[102px]"
+              multiple id='attributes' 
+              value={selectedAttributes} 
+              onChange={handleAttributeChange}
+            >
+              <option value="CaseID">CaseID</option>
+              <option value="EventCode">EventCode</option>
+              <option value="EventName">EventName</option>
+              <option value="MMWRYear">MMWRYear</option>
+              <option value="MMWRWeek">MMWRWeek</option>
+            </select>
+
```

Comment:
CaseID should not be an option. You are also missing the CountyReporting, CaseClassStatus, Sex, BirthDate, Age, AgeType, Race, and Ethnicity fields.